bioblend/_tests/TestGalaxyLibraries.py

```diff
@@ -12,7 +12,10 @@ class TestGalaxyLibraries(GalaxyTestBase.GalaxyTestBase):
     def setUp(self):
         super(TestGalaxyLibraries, self).setUp()
         self.name = 'automated test library'
+        self.deleted_name = 'deleted test library'
         self.library = self.gi.libraries.create_library(self.name, description='automated test', synopsis='automated test synopsis')
+        self.deleted_library = self.gi.libraries.create_library(self.deleted_name, description='a deleted library', synopsis='automated test synopsis')
+        self.gi.libraries.delete_library(self.deleted_library['id'])
```

Comment:
I'd move these 3 new lines to the `test_get_libraries()` method, since the deleted library is used only for that test.